backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/Linker.kt

```diff
@@ -137,6 +137,7 @@ internal class Linker(val context: Context) {
         }
 
         val needsProfileLibrary = context.coverage.enabled
+        val mimallocEnabled = config.get(KonanConfigKeys.ALLOCATION_MODE) == "mimalloc"
```

Comment:
There's an edge case: if you enabled `mimalloc` via command line on a target that doesn't support it, we would complain and link standard allocator instead: https://github.com/JetBrains/kotlin-native/blob/a710a478a522b1ba03e66d3dd85f2d5a8213e83d/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/KonanConfig.kt#L120

Reply:
Good point.